src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/getUserById';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
```

Comment:
[CODE KNOWLEDGE] - You are using `getUserById` to fetch data inside a component. This can lead to performance issues if `TodoInfo` is rendered multiple times, as it would trigger the `getUserById` function each time. Consider lifting this logic up to a parent component or using a state management library to avoid unnecessary calls.